src/main/webapp/script.js

```diff
@@ -128,13 +137,36 @@ function createOmdbItemCard(omdbItem) {
   return card;
 }
 
+/**
+ * Adds a callback to the item submission button to send an omdbItem to the
+ * ItemSubmissionServlet using a Post request.
+ *
+ * @param { jQuery } submissionButton - the button in the dialog used to submit
+ *     an Entertainment Item
+ * @param { JSON } omdbItem - item that will be sent to the
+ *     ItemSubmissionServlet through a Post request
+ */
+function addItemSubmissionCallback(submissionButton, omdbItem) {
+  submissionButton.click(function() {
+    $.post('/item-submission', omdbItem)
+        .done(function() {
+          // If the item gets added, the page needs to reload to see the
+          // changes.
+          window.location.reload();
+        })
+        .fail(function() {
+          console.log('Failed to submit entertainment item!');
+        });
+  });
+}
```

Comment:
It feels a little unnecessary to pass in the submissionButton to the function: 

Something like this flows a little better:
```suggestion
function addItem(omdbItem) {
    $.post('/item-submission', omdbItem)
        .done(function() {
          // If the item gets added, the page needs to reload to see the
          // changes.
          window.location.reload();
        })
        .fail(function() {
          console.log('Failed to submit entertainment item!');
        });
}
```

Then you can replace ` addItemSubmissionCallback(itemSubmissionButton, omdbItem);` with
`itemSubmissionButton.click(addItem);`

Reply:
I simplified the submit button callback function in last commit.